libctru/include/3ds/services/frd.h

```diff
@@ -119,7 +119,7 @@ Result FRD_GetMyFavoriteGame(u64 *titleId);
  * @brief Gets the current user's comment on their friend profile.
  * @param comment Pointer to write the current user's comment to.
  */
-Result FRD_GetMyComment(u16 *comment);
+Result FRD_GetMyComment(u8 *comment);
```

Comment:
Should be char*, and should have a parameter to describe the output buffer size (and use it).

Reply:
>  should have a parameter to describe the output buffer size
Hmmm, just strlen it?

Reply:
I meant, the total capacity of the buffer the user provides to the function.